### ISSUE TITLE
onet.pl

### ISSUE THREAD

**Issue by Jurigag** [reek/anti-adblock-killer#1881]
Im using opera 39 with native adblocker(installed filter list) and tampermonkey. From today i think it there is anti-adblock like this:

http://x3.cdn03.imgwykop.pl/c3201142/comment_75FthpzJdJAtcWLYHfH4xqVvnsIcv0w0,w400.jpg

And page is removing content from itself.

**Comment by xxcriticxx** [comment]
podaj link do strony

dodam do polskiego forum

**Comment by Jurigag** [comment]
www.onet.pl

**Comment by xxcriticxx** [comment]
i don't see any adblock on main page  

please provide better link

**Comment by Jurigag** [comment]
I see on main page.

**Comment by xxcriticxx** [comment]
please list your setup

**Comment by X01X012013** [comment]
I don't see the block screen neither. 
Opera, ABP, Tampermonkey.

**Comment by xxcriticxx** [comment]
maybe no userscript

**Comment by Jurigag** [comment]
As I wrote earlier, opera, native built-in adblocker, ghostery, userscript
is there, as well as filter lists in Opera.
6 sie 2016 16:00 "xxcriticxx" <notifications@github.com> napisał(a):

maybe no userscript

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/reek/anti-adblock-killer/issues/1881#issuecomment-238024618>,
or mute the thread

**Comment by xxcriticxx** [comment]
turn ghostery and test

**Comment by osiu97** [comment]
i checked it myself. It only shows once in a few refreshes but it's there and it's annoying ;)

**Comment by xxcriticxx** [comment]
@osiu97 whats your setup?

when you find it can you take screen shoot of your logger

**Comment by eventhorizonbreaker** [comment]
Hi,

This happens to me since 2 days on permanent basis. Main portal of www.onet.pl displays just upper part while majority of it does not appear and the picture indicated by Jurigag is displayed. I haven't used any anti-adblock tool so far so I installed the anti-block-killer and it did not help.

My setup is:
Chrome: 52.0.2743.116 m (64-bit)
uBlock Origin: 1.8.4
Anti-Adblock Killer | Reek subscribed
Tampermonkey 1.4 installed
Anti-Adblock Killer | Reek 9.8 script installed

Is there anything I can check? Any log file to see why it fails?

**Comment by szymon1118** [comment]
Hi,

Yesterday I found a solution and this script is blocking bottom part of the site: "ty0lm9uz.onet.pl/YnVpbGQvZGxBcGkvZGwubGliLm1pbg%3D%3D.js". Personally I'm using Adblock Plus so I made a rule for it: "||ty0lm9uz.onet.pl/YnVpbGQvZGxBcGkvZGwubGliLm1pbg%3D%3D.js".

Hope I helped someone :)

**Comment by eventhorizonbreaker** [comment]
Yes, that was it indeed, thanks. Let's hope they won't use random chars every week.

**Comment by eventhorizonbreaker** [comment]
Today js file name changed. I modified the filter to "||ty0lm9uz.onet.pl/*.js"

**Comment by szymon1118** [comment]
Correction: "||ty0lm9uz.onet.pl/*%3D%3D.js". Moreover a script name is encoded with base64.

**Comment by eventhorizonbreaker** [comment]
Well... it changed again and there is no longer "==" at the end. It uses however the same path encoded in the random string. Currently I use this one: "||ty0lm9uz.onet.pl/*YnVpbGQvZGxBcGkv*.js".

**Comment by Jurigag** [comment]
Well they again changed it, i just added:

```
||ty0lm9uz.onet.pl/*
```

**Comment by mapx-** [comment]
or add
`/http://[a-z0-9]{8}.onet.pl/$script`

**Comment by Jurigag** [comment]
Im guessing they reading this issue :D

**Comment by eventhorizonbreaker** [comment]
Indeed. ||grfjwq.onet.pl/YnthNUxfTTlyfXVpbGQvZGxBcGkve3E3UlZRb31kbC5saWIubWluLmpz at least for now.

**Comment by niutech** [comment]
In the source there is `<script id="adsinit" data-original="http://grfjwq.onet.pl/YnthNUxfTTlyfXVpbGQvZGxBcGkve3E3UlZRb31kbC5saWIubWluLmpz" type="text/javascript">`, so blocking #adsinit regardless of the URL should be enough, shouldn't it?

**Comment by eventhorizonbreaker** [comment]
I'm not that good with filters. I see a way to hide elements by id but not block them. What would be the syntax to block instead of hiding them (which does not work the way I tried)?

**Comment by szymon1118** [comment]
They (https://github.com/MajkiIT/polish-ads-filter) blocked onet antiadblock few weeks ago...

**Comment by niutech** [comment]
There is a rule: `/:\/\/[a-zA-Z0-9]{6,8}\.onet.pl\//$script,xmlhttprequest,domain=www.onet.pl` - we will see how long it works.

**Comment by Jurigag** [comment]
Hell man, for like 100% they reading this :D

**Comment by Jurigag** [comment]
`/:\/\/[a-zA-Z0-9]{6,9}\.onet.pl\//$script,xmlhttprequest,domain=www.onet.pl` no longer works, any idea ?

**Comment by eventhorizonbreaker** [comment]
You can always do the dirty trick temporarily. This time it is:
||sgqcvfjvr.onet.pl/YnthN0xfTTlyfXVpbGQvZGxBcGkve3E3UlZRb31kbC5saWIubWluLmpz

**Comment by Jurigag** [comment]
So my rule should, why it isn't ? :x

**Comment by szymon1118** [comment]
I already mentioned that when you add their rules (https://github.com/MajkiIT/polish-ads-filter) from here: https://www.certyficate.it/adblock/ you can get rid of onet anti-adblock.

**Comment by Jurigag** [comment]
Which exactly file i need to add to opera ? I tried adblock.txt and not working.

**Comment by szymon1118** [comment]
Sorry but I really don't know why it's not working for you :D. Personally I'm using chrome with ABP, ublock with filters below:
![image](https://cloud.githubusercontent.com/assets/6980624/19363326/57e3ec1a-918a-11e6-9063-5b984f11a723.png)

ublock: AAK and polish-ads-filter

**Comment by Jurigag** [comment]
I installed both lists from https://github.com/MajkiIT/polish-ads-filter and not working.

**Comment by Jurigag** [comment]
What a hell, i cleared local storage and cookies and it start working.
